package.json

```diff
@@ -6,8 +6,7 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
     "start-zh": "docusaurus start --locale zh-CN",
-    "build": "npm run team && docusaurus build",
-    "team": "node ./scripts/team.js",
```

Comment:
Keep the sync team command and change it to sync-team
`"sync-team": "node ..."`